x-pack/legacy/plugins/siem/public/components/link_to/redirect_to_case.tsx

```diff
@@ -30,7 +31,11 @@ export const RedirectToConfigureCasesPage = () => (
 
 const baseCaseUrl = `#/link-to/${SiemPageName.case}`;
 
-export const getCaseUrl = () => baseCaseUrl;
-export const getCaseDetailsUrl = (detailName: string) => `${baseCaseUrl}/${detailName}`;
-export const getCreateCaseUrl = () => `${baseCaseUrl}/create`;
-export const getConfigureCasesUrl = () => `${baseCaseUrl}/configure`;
+export const getCaseUrl = (search: string | null) =>
```

Comment:
considering change search into an optional param.

Reply:
🙏  The same reasoning for keeping it non-optional, as discussed above in https://github.com/elastic/kibana/pull/60993#discussion_r396836926 is, imo, applicable here.